libnavui-androidauto/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Unreleased
 #### Features
+- Added support for MUTCD and Vienna designs in the Speed limit widget. [#6275](https://github.com/mapbox/mapbox-navigation-android/pull/6275)
+- Added options to customize behavior of the Speed limit widget. [#6275](https://github.com/mapbox/mapbox-navigation-android/pull/6275)
+
```

Comment:
wonder if this space is the reason changelog fails 🤔

Reply:
never mind this one is passing https://github.com/mapbox/mapbox-navigation-android/pull/6263